docs/personalization/integrate_recommendation_service.md

```diff
@@ -151,6 +151,62 @@ if ($recommendations && isset($recommendations->recommendationResponseList)) {
 curl_close($curl);
 ```
 
+## Send emails with recommendations
+
+By using [email triggers]([[= user_doc =]]/personalization/triggers.md), your organization can send emails to individual visitors, for example, to invite them to return to the website or remind them of items abandoned in a cart.
+
+Email triggers are push messages with recommendations.
+Specific conditions, such as the time that must pass before email start being sent, content types and attributes to be included in a response, or a number of repetitions, are first defined based on an interview between you and Ibexa.
```

Comment:
Specific conditions, such as the time that must pass before email start being sent -> passive voice and weird phrasing, did you mean start of email communication, newsletter chain?
content types -> Content Types
Please update the sentence, maybe a list would help. It is hard to follow and understand.